src/pal/pal_linux.h

```diff
@@ -26,7 +26,7 @@ namespace snmalloc
     static constexpr uint64_t pal_features = PALPOSIX::pal_features;
 
     static constexpr size_t page_size =
-      Aal::aal_name == PowerPC ? 0x10000 : 0x1000;
+      (Aal::aal_name == PowerPC ? 0x10000 : (Aal::aal_name == Sparc ? Aal::smallest_page_size : 0x1000));
```

Comment:
Does this actually need to be guarded on `Sparc`?  I think we should use the smallest page size or the larger value on PowerPC.

```suggestion
      (Aal::aal_name == PowerPC ? 0x10000 : Aal::smallest_page_size);
```